contracts/staking/contracts/src/sys/MixinFinalizer.sol

```diff
@@ -38,56 +38,54 @@ contract MixinFinalizer is
     ///      If there were no active pools in the closing epoch, the epoch
     ///      will be instantly finalized here. Otherwise, `finalizePool()`
     ///      should be called on each active pool afterwards.
-    /// @return poolsRemaining The number of unfinalized pools.
+    /// @return poolsToFinalize The number of unfinalized pools.
     function endEpoch()
```

Comment:
I'm definitely onboard with this change. The term "finalize" is definitely overloaded in the codebase; for example, we also have "unfinalized stake" and "unfinalized fees". 

A change like this would be very widespread (see similar #2264). I think it would be too much to tack onto this PR but I'm open to trying it in a new one, given there's time.